targets/posix/nanoCLR/nanoCLR_native_posix.cpp

```
@@ -0,0 +1,205 @@
+#include <cstdlib>
+#include <cstring>
+
+// ── Forward declarations of impls that live in CLRStartup.cpp ───────────────
+HRESULT nanoCLR_LoadAssemblyImpl(const wchar_t *name, const uint8_t *data, size_t size);
+HRESULT nanoCLR_LoadAssembliesSetImpl(const uint8_t *data, size_t size);
+HRESULT nanoCLR_ResolveImpl();
+void nanoCLR_SetConfigureCallbackImpl(ConfigureRuntimeCallback cb);
+
+// ── Callback globals ────────────────────────────────────────────────────────
+
+DebugPrintCallback gDebugPrintCallback = nullptr;
```

Clang (14.0.6)

[warning] variable 'gDebugPrintCallback' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)